railties/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Fix `ActiveRecord::Persistence::ClassMethods#insert_all!` not supporting `unique_by` option.
```

Comment:
This changelog should be in activerecord/CHANGELOG.md not in Railties.

I also think it should say "`ActiveRecord::Persistence::ClassMethods#insert_all!` now supports the `unique_by` option" and include an example of usage.

Reply:
Thanks for noting